src/main/java/hudson/plugins/jira/JiraFolderProperty.java

```diff
@@ -54,18 +54,11 @@ public void setSites(List<JiraSite> sites) {
         this.sites = sites;
     }
 
+    /**
+     * @deprecated use {@link JiraSite#getSitesFromFolders(ItemGroup)}
+     */
     public static List<JiraSite> getSitesFromFolders(ItemGroup itemGroup) {
```

Comment:
@Deprecated

Reply:
the annotation doesn't accept any comment or description (a bit useless and redundant?)

Reply:
https://stackoverflow.com/questions/20674135/deprecated-vs-deprecated#:~:text=%40Deprecated%20is%20an%20annotation%20that,provide%20documentation%20about%20the%20deprecation.

Reply:
yup for sure it doesn't hurt.